google-ads-bom/build.gradle

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * A Bill of Materials (BOM) for the Google Ads API client library for Java.
+ *
+ * This BOM ensures that all Google Ads API library artifacts and their
+ * transitive dependencies are using compatible versions.
+ *
+ * To use this BOM, add the following to the dependencyManagement section of your
+ * pom.xml file:
+ *
+ * <dependencyManagement>
+ *   <dependencies>
+ *     <dependency>
+ *       <groupId>com.google.api-ads</groupId>
+ *       <artifactId>google-ads-bom</artifactId>
+ *       <version>REPLACE-WITH-CURRENT-VERSION</version>
+ *       <type>pom</type>
+ *       <scope>import</scope>
+ *     </dependency>
+ *   </dependencies>
+ * </dependencyManagement>
+ *
+ * Then, add dependencies on the Google Ads API library artifacts without
+ * specifying their versions.
```

Comment:
Should we provide an example of a version-less dependency here for additional clarity? I know developers should know what it looks like, but you never know :)